SoftLayer/CLI/routes.py

```diff
@@ -79,6 +79,7 @@
     ('block:volume-count', 'SoftLayer.CLI.block.count:cli'),
     ('block:volume-detail', 'SoftLayer.CLI.block.detail:cli'),
     ('block:volume-duplicate', 'SoftLayer.CLI.block.duplicate:cli'),
+    ('block:volume-modify', 'SoftLayer.CLI.block.modify:cli'),
```

Comment:
This is minor, but it would be ideal if this line was moved just below the line for 'volume-list', so that the commands remain in alphabetical order.